framework/src/utils/PerfGraph.C

```diff
@@ -158,29 +229,67 @@ PerfGraph::push(const PerfID id)
     mooseError("PerfGraph is out of stack space!");
 
   _stack[_current_position] = new_node;
+
+  // Add this to the exection list
+  if ((_live_print_active || _live_print_all) && _pid == 0 &&
```

Comment:
Doesn't `_live_print_all` here circumvent where you purposely disabled  it because of the clobbering?

Reply:
Yes - `_live_print_all` should trump everything.